### ISSUE TITLE
"compute_bounds" Method ("interface.py") Improvement

### ISSUE THREAD

**Issue by lelaus** [newcleo-dev-team/lbh15#52]
- [1] before exploiting the results of the minimization procedure, check the value of the `OptimizeResult.success` boolean flag;
- [2] about the rounding operations close to the validity range extrema:
- [2.A] check whether they are useful, otherwise delete them;
- [2.B] in case they remain:
- [2.B.1] apply them to both minimum and maximum values of the validity range for each extremum value;
- [2.B.2] check the sign of the if statement conditions;
- [2.B.3] put the tolerance value within the commons;
- [2.B.4] try to encapsulate the checks within an external private function, thus implementing them only once;
- [3] about the function to focus on for finding the maximum:
- [3.A] change it to the opposite of the original correlation;
- [3.B] put it within the "minimize_scalar()" function arguments directly, without any external def;

**Comment by lelaus** [comment]
Some thoughts about the rounding operations close to the validity range extrema:
1. checking whether the temperature value is within the validity range and close to one of its extrema, thus changing its value to that of one of the extrema themselves seems useless; why changing the temperature value?
2. checking whether the temperature value is outside the validity range and close to its extrema, thus changing its value to that of one of the extrema themselves can be incorrect; this depends on the accuracy related to both the correlation validity range (not always available) and the temperature value resulting from the application using the lbh15 library.
To remove the rounding operations close to the validity range extrema seems the direct consequence of the above points.

**Comment by lelaus** [comment]
Within tests, the "assertAlmostEqual" method uses decimal places. It should be more effective to refer to the most significant digits.

**Comment by lelaus** [comment]
Alternatively, rounding the reference values to a specified number of digits should improve testing operations as well.

**Comment by panDanieleN** [comment]
Merged into master.

Updated `properties_bounds.json` file in tests directory.

No need for documentation update.
